src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+				minLabel={componentObj.minLabel}
+				maxLabel={componentObj.maxLabel}
+				light={componentObj.light} />
+			</div>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ASlider,
+	settingsUI: ASliderSettingsUI,
+	codeUI: ASliderCodeUI,
+	render: ({ componentObj, select, remove, selected }) => <ASlider
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected} />,
+	keywords: ['slider'],
+	name: 'Slider',
+	type: 'slider',
+	defaultComponentObj: {
+		type: 'slider',
+		labelText: 'Slider label',
+		min: 0,
+		max: 100
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Label = "${json.labelText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled ? json.disabled : false};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = ${json.value ? json.value : json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Min = ${json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Max = ${json.max};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MinLabel = "${json.minLabel ? json.minLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MaxLabel = "${json.maxLabel ? json.maxLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Step = ${json.step ? json.step : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}StepMultiplier = ${json.stepMultiplier ? json.stepMultiplier : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}" `,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+			imports: ['SliderModule'],
+			otherImports: () => 'import { CommonModule } from "@angular/common"',
+			code: ({ json }) => {
+				return `<ibm-slider
+					[label]="${nameStringToVariableString(json.codeContext?.name)}Label"
+					[min]="${nameStringToVariableString(json.codeContext?.name)}Min"
+					[max]="${nameStringToVariableString(json.codeContext?.name)}Max"
+					[step]="${nameStringToVariableString(json.codeContext?.name)}Step"
+					[value]="${nameStringToVariableString(json.codeContext?.name)}Value"
+					[shiftMultiplier]="${nameStringToVariableString(json.codeContext?.name)}StepMultiplier"
+					[disabled]="${nameStringToVariableString(json.codeContext?.name)}Disabled"
+					(valueChange)="${nameStringToVariableString(json.codeContext?.name)}ValueChange.emit($event.value)">
+					<span minLabel>{{${nameStringToVariableString(json.codeContext?.name)}MinLabel}}</span>
+					<span maxLabel>{{${nameStringToVariableString(json.codeContext?.name)}MaxLabel}}</span>
+                    <input theme="${nameStringToVariableString(json.codeContext?.name)}Theme" 
```

Comment:
Theme should be binded, otherwise the string is being assigned to the `theme` prop.